### ISSUE TITLE
Add Bridge Wallet to Ethereum wallets

### ISSUE THREAD

**Issue by yannggg** [ethereum/ethereum-org-website#1458]
**Does the wallet have fiat on-ramps?**

Yes, Bridge Wallet currently has a bank-transfer on-ramp from CHF/EUR/USD/GBP to ETH/DAI/USDT/USDC/BTC. We are currently working on adding more payment methods. On/off-ramp network fees are currently subsidized, but this will end in the coming months.

**Does the wallet allow users to explore dapps?**

Yes, Bridge Wallet integrates WalletConnect.

**Does the wallet have integrated defi/financial tools?**

Yes, Bridge Wallet integrates Uniswap (v1). We are currently working on adding Uniswap v2 support.

**Can a user withdraw to their card?**

Users can currently cash-out ETH/DAI/USDT/USDC/BTC in 40+ fiat currencies to their bank account. We are in negociation with a potential partner to enable card withdrawals as well.

**Does the wallet offer limits protection?**

Not at the moment, we haven't seen that feature requested in our community.

**Does the wallet allow high-volume purchases?**

Yes, there is no minimum or maximum purchase with Bridge Wallet. High-volume purchases might only request additional KYC/AML information before going through, a process which is fully performed in-house as we are an authorized financial intermediary in Switzerland.

**Does the wallet have an integrated token swap?**

Yes, users can swap any ERC-20 token in the app via Uniswap or via Bridge Wallet's own P2P swap order book.

**Is the wallet a multi-signature wallet?**

No

**Wallet title**

Bridge Wallet

**Wallet description**

Bridge Wallet is the non-custodial mobile wallet to manage crypto purchases and cash-outs the easy way.

**Wallet logo**

https://www.mtpelerin.com/images/bridge-icon.svg

**Brand color**

#001e32

**Wallet link**

https://www.mtpelerin.com/bridge-wallet

**Comment by GeorgeTrotter** [comment]
Hey @yannggg! We're reviewing Bridge Wallet's submission for listing on ethereum.org according to our [listing policy.](https://ethereum.org/en/contributing/adding-products/) It seems from [this tweet](https://twitter.com/mtpelerin/status/1268498746108792833) that Bridge Wallet went live on June 4th, 2020, which would be less than six months. Can you send us something to confirm that Bridge Wallet has been live from end of March or before?

Can you also let us know how we can confirm your last audit of the wallet?

Thank you!!

**Comment by yannggg** [comment]
Hey @GeorgeTrotter! It hasn't been externally audited yet. For the live date, we've soft launched the wallet before that Tweet, here is a screenshot of the App Store dev dashboard's release history to show that is has been live for more than 6 months:

[Screenshot](https://www.mtpelerin.com/images/screencapture-appstoreconnect.png)

**Comment by GeorgeTrotter** [comment]
Awesome! Thanks for that screenshot. We do need the wallets listed on ethereum.org to be externally audited. If you can provide confirmation of your audit down the road we will absolutely review Bridge Wallet for listing!

**Comment by GeorgeTrotter** [comment]
Hey @yannggg! We're going to close this issue out for now. Feel free to open a new issue in the future once your audit is complete! Let us know if you have any questions.
